lib/pascaltriangle.rb

```diff
@@ -0,0 +1,26 @@
+class Triangle
+
+  def initialize
+    @pascals_row = []
+  end
+
+  def pascal(row_number)
+    if row_number ==  0
+      @pascals_row << [1]
+    else
+      @pascals_row << (0..row_number).map { |i| binomial(row_number, i) }
+    end
+  end
+
+  private
+  def factorial(n)
+    (1..n).inject(:*)
+  end
+
+  private
```

Comment:
@zzokki81 once you've declared a method private all subsequent methods are private too until you specify `protected` or `public`.